app/src/behaviors/behavior_soft_off.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
```

Comment:
```suggestion
 * Copyright (c) 2023 The ZMK Contributors
```
unless you started writing this file in 2020?